libraries/testing/contracts/eosio.bios/eosio.bios.cpp

```diff
@@ -74,12 +74,12 @@ void bios::setfinalizer( const finalizer_policy& finalizer_policy ) {
       check(eosio::bls_pop_verify(pk, signature), "proof of possession failed");
 
       std::vector<char> pk_vector(pk.begin(), pk.end());
-      abi_finalizer_policy.finalizers.emplace_back(eosio::abi_finalizer_authority{f.description, f.weight, std::move(pk_vector)});
+      fin_policy.finalizers.emplace_back(eosio::finalizer_authority{f.description, f.weight, std::move(pk_vector)});
    }
 
    check(finalizer_policy.threshold > weight_sum / 2, "finalizer policy threshold must be greater than half of the sum of the weights");
 
-   set_finalizers(std::move(abi_finalizer_policy));
+   set_finalizers(std::move(fin_policy));
```

Comment:
No, CDT provides a `set_finalizers` that hides that from users. Similarly CDT hides the format for the `set_proposed_producers_ex`. This is nice because it means no update to reference contracts needed.
See for example https://github.com/AntelopeIO/cdt/blob/main/libraries/eosiolib/contracts/eosio/privileged.hpp#L233